Tests/SwiftProtobufTests/Test_Enum.swift

```diff
@@ -40,6 +41,23 @@ class Test_Enum: XCTestCase, PBTestHelpers {
         }
     }
 
+    func testJSONIgnoreUnknownValues() {
```

Comment:
I think I'd rather have this test case in `Test_JSONDecodingOptions.swift`

Reply:
Done.